### PR TITLE
React State 2: Table Reservation Challenge

### PR DESCRIPTION
**Pull request by PReichetanz**
### 🔎 Review

Review via [🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/react-state-2-table-reservation/sessions/react-state-2/table-reservation?file=/README.md) or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/react-state-2-table-reservation/sessions/react-state-2/table-reservation -i
```

### ✨ Copyable Markdown for Curriculum

````md
Solve this challenge on
[🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/react-state-2/table-reservation?file=/README.md)
or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/react-state-2/table-reservation -i
```
````

refers to neuefische/web-curriculum-new-format#196

### CODE REVIEW

File: sessions/react-state-2/table-reservation/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+You can use the following hints as guideline:
+
+- Move the `people` state to the `App` component.
+- In the `App` component, you want to have two helper functions to `handle` adding and subtracting a person from the `people` state.
+- Pass your `handle` functions to the `Counter` component:
+  - use one prop for each of them;
+  - the prop should have the same name as the `handle` function, but should start with `on` instead of `handle` (Hint: `onDelete={handleDelete}`).
+- In the `Counter` component
+  - receive the two newly created props and
+  - pass them to the correct button's `onClick` prop. Make sure **not** to use an anonymous function here.
```
Comment (djfarly) [inline sessions/react-state-2/table-reservation/README.md:25]: why?

Comment (djfarly) [reply sessions/react-state-2/table-reservation/README.md:25]: Finde es fein, das mit inline functions zu lösen:
```js
      <Counter
        onIncrement={() => setPeople(people + 1)}
        onDecrement={() => setPeople(people - 1)}
      />
```

**Review by djfarly — APPROVED** [review]
besides the fact that i think it's also fine to solve using anon functions: LGTM! :)

File: sessions/react-state-2/table-reservation/README.md (hunk 1 above)
Comment (PReichetanz) [reply sessions/react-state-2/table-reservation/README.md:25]: I see the point. I think it's then better to delete the second hint about the anonymous function, like so:
```suggestion
  - pass them to the correct button's `onClick` prop.
```

**Pull request merged**